packages/api/lambdas/index-from-database.js

```diff
@@ -112,59 +202,85 @@ async function indexFromDatabase(event) {
   await Promise.all([
     indexModel({
       esClient,
-      tableName: tables.collectionsTable,
       esIndex,
       indexFn: indexer.indexCollection,
       limitEsRequests,
+      postgresModel: new CollectionPgModel(),
```

Comment:
I wonder if we should be doing a p-map with limited concurrency rather than a Promise.all of indexing all the tables at once. As long as we're limiting the concurrency of requests within indexing for each table, we should be okay memory wise, but I wonder if we will easily overwhelm our RDS serverless databases?

Reply:
Hmm.  That's a really good point, I'll see what I can do with that.